server/utils/comments/comments.util.js

```diff
@@ -0,0 +1,21 @@
+import validator from 'validator';
```

Comment:
I like the fact that your variable naming is descriptive which makes anyone reading the code to understand the function of the variable.